src/app/util/ClientMonitoringRegistrationTable.cpp

```diff
@@ -17,59 +17,71 @@
 
 #include "ClientMonitoringRegistrationTable.h"
 
-namespace chip {
+#include <lib/support/DefaultStorageKeyAllocator.h>
 
-/**********************************************************
- * Attributes Definition
- *********************************************************/
+namespace chip {
 
 /**********************************************************
  * ClientMonitoringRegistrationTable Implementation
  *********************************************************/
 
-ClientMonitoringRegistrationTable::ClientMonitoringRegistrationTable(FabricIndex fabricIndex)
-{
-    this->LoadFromStorage(fabricIndex);
-}
+ClientMonitoringRegistrationTable::ClientMonitoringRegistrationTable(PersistentStorageDelegate & storage) : mStorage(storage) {}
 
-void ClientMonitoringRegistrationTable::LoadFromStorage(FabricIndex fabricIndex)
+/**
```

Comment:
I think we typically put doxygen comments in header files so that one doesn't need to look into implementation to learn details. I suggest moving it there for consistency.

Reply:
moved all the comments